documentation/release-notes/topics/version_4.rst

```diff
@@ -2,6 +2,51 @@
 Version 4
 #########
 
+*******************
+Hyperview 4.2 (TBD)
+*******************
+This section covers significant changes and bug fixes in Hyperview 4.2.x since version 4.1.X
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+=======================================
+Enhanced Licensed Feature: Connectivity
+=======================================
+Port management has been greatly improved:
+
+- The port name template is editable when adding new ports from the Information -> Network Ports page
+- The port name template is editable when adding new ports from the Layout page of network devices and patch panels
+- Port names are now editable in bulk from the Information -> Network Ports page. This will allow bulk changes to port names to align with internal or manufacturer port naming conventions
```

Comment:
We might also want to mention the Layout page of network devices and patch panels.